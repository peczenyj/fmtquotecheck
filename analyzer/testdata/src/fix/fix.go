package fix

import (
	"fmt"
)

func Setup() {
	fmt.Printf(
		"just printing from '%s' with '%s'", // want "explicit single-quoted '%s' should be replaced by `%q` in fmt.Printf"
		"function",
		"style",
	)
}
