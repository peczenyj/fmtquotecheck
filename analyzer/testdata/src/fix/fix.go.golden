package fix

import (
	"fmt"
)

func Setup() {
	fmt.Printf(
		"just printing from %q with %q", // want "explicit single-quoted '%s' should be replaced by `%q` in fmt.Printf"
		"function",
		"style",
	)
}